README.adoc

```diff
@@ -40,20 +40,21 @@ Open Liberty. If you run the application, Maven creates an Open Liberty server a
 the application on it. The application displays a simple web page with a link that, when 
 clicked, calls the servlet to return a simple response of `Hello! How are you today?`.
 
-One benefit of using Maven is that, once you have defined the details of the project and 
-any dependencies it has, Maven automatically handles downloading and installing any and 
-all of the dependencies.
+One benefit of using a build tool like Maven is that you can define the details of the project and any dependencies it has, and then Maven automatically downloads and installs the dependencies.
+Another benefit of using Maven is that it can run repeatable, automated tests on the application. 
+You could, of course, test your application manually by starting a server and pointing a web browser at the application URL. 
+Automated tests are a much better approach because you can easily rerun the same tests each time the application is built. 
+If the tests don't pass after you've made a change to the application, the build fails, and you know that you introduced a regression that requires a fix to your code.
 
-Another benefit of using Maven is that it can run automated tests on the application after 
-building it. You could, of course, test your application manually by starting a server 
-and pointing a web browser at the application URL. Automated tests are a much better 
-approach though because Maven can easily re-run the same tests each time it builds the 
-application. If the tests don't pass after you've made a change to the application, the 
-build fails so you know that you need to fix your code.
+Choosing a build tool often comes down to personal or organizational preference, but you might choose to use Maven for several reasons. 
+Maven defines its builds using XML, which is probably familiar to you already. 
+As a mature, commonly used build tool, Maven probably integrates with whichever IDE you prefer to use. 
+Maven also has an extensive plugin library which offers a variety of ways to quickly customize your build. 
+Maven can be a good choice if your team is already familiar with it. 
 
-You will create a Maven build definition file (`pom.xml`) for the web application project 
-and use it to build the web application. You will then create a simple, automated test 
-and configure Maven to run it after building the application.
+You will create a Maven build definition file called a `pom.xml` file, which stands for 
+project object model, and use it to build your web application. You will then create a simple, 
+automated test and configure Maven to run automatically run the test.
```

Comment:
Suggested update

>You will create a Maven build definition file that's called a `pom.xml` file, which stands for 
Project Object Model, and use it to build your web application. You will then create a simple, 
automated test and configure Maven to automatically run the test.